packages/flutter_tools/test/integration.shard/isolated/native_assets_test_utils.dart

```diff
@@ -0,0 +1,279 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:file/file.dart';
+import 'package:file_testing/file_testing.dart';
+
+import '../../src/common.dart';
+import '../test_utils.dart' show ProcessResultMatcher, fileSystem;
+import '../transition_test_utils.dart';
+
+Future<Directory> createTestProject(String packageName, Directory tempDirectory) async {
+  final ProcessResult result = processManager.runSync(
+    <String>[
+      flutterBin,
+      'create',
+      '--no-pub',
+      '--template=package_ffi',
+      packageName,
+    ],
+    workingDirectory: tempDirectory.path,
+  );
+  if (result.exitCode != 0) {
+    throw Exception(
+      'flutter create failed: ${result.exitCode}\n${result.stderr}\n${result.stdout}',
+    );
+  }
+
+  final Directory packageDirectory = tempDirectory.childDirectory(packageName);
+
+  // No platform-specific boilerplate files.
+  expect(packageDirectory.childDirectory('android/'), isNot(exists));
```

Comment:
Since this is using `childDirectory` can we remove the trailing `/` and have the test have the same meaning? If so lets do that so that we dont have any issues with windows folder paths.